test/module/irohad/ametsuchi/storage_init_test.cpp

```diff
@@ -15,13 +15,17 @@ using namespace iroha::expected;
 class StorageInitTest : public ::testing::Test {
  public:
   StorageInitTest() {
-    std::string env_var;
+    auto pg_host = std::getenv("IROHA_POSTGRES_HOST");
```

Comment:
Please use the existing method instead
https://github.com/hyperledger/iroha/blob/25c90357a796034cbee688a45cb55d1b2163bd08/test/framework/config_helper.cpp#L24-L37

Reply:
how it is better?

Reply:
Because the logic is the same, so no need to maintain it in two different methods.